Addressbook/src/main.sh

```diff
@@ -0,0 +1,119 @@
+#!/bin/ash
+
+source addressbook.sh
+
+echo "Welcome!"
+echo
+
+usage_menu(){
+	echo "Commands:"
```

Comment:
Tabs and spaces are mixed throughout the files, sometimes on a single line. Please, please don't do that ever.